lib/phoenix/channel/transport.ex

```diff
@@ -206,4 +206,34 @@ defmodule Phoenix.Channel.Transport do
     end
     :ok
   end
+
+  def origin_allowed?(nil, _) do
```

Comment:
We should document this. The idea is that: if the client does not send an origin, we are not going to check for it. However, this is not a problem with browsers because they all send the origin, correct?